Controllers/getArticleId.controller.js

```diff
@@ -0,0 +1,11 @@
+const { getArticleById } = require("../Models/getArticleById.model");
+
+exports.getArticle = async (req, res) => {
+  try {
+    const { article_id } = req.params;
+    const article = await getArticleById(article_id);
+    res.status(200).json({ article });
+  } catch (error) {
+    res.status(404).json({ message: error.message });
```

Comment:
we want to use custom error handlers to send the errors for us - have a look at the NC Notes on error handling to find out how to add your error handling middleware